docs/api/merchants.md

```
@@ -0,0 +1,117 @@
+  "name": "Whole Foods",
+  "type": "FamilyMerchant",
+  "created_at": "2024-01-15T10:30:00Z",
+  "updated_at": "2024-01-15T10:30:00Z"
+}
+```
+
+## Listing merchants
+
+Example request:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)